doc/read-the-docs-site/tutorials/english-auction-example.rst

```diff
@@ -0,0 +1,344 @@
+The two inputs are:
+
+* the auction EUTXO
+* Charlie's bid of 200 Ada
+
+The two outputs are:
+
+* the updated auction EUTXO
+* a EUTXO that returns Bob's bid of 100 Ada
+
+.. note::
+    To be more clear, technically, the auction EUTXO is not getting updated because nothing ever changes.
+    Instead, what really happens is that the old auction EUTXO is spent and a new one is created.
+    In a way this may feel like the auction EUTXO is getting updated, but that isn't truly accurate.
+
++--------------------------+-----------------------------------------+
+| 2nd bid inputs           | 2nd bid outputs                         |
++==========================+=========================================+
+| The auction EUTXO        | Newly created unspent transaction       |
++--------------------------+-----------------------------------------+
+| Charlie's bid of 200 Ada | EUTXO that returns Bob's bid of 100 Ada |
++--------------------------+-----------------------------------------+
+
+Bid redeemer logic
+~~~~~~~~~~~~~~~~~~~~~
+
+This time we again use the `Bid` redeemer. The script has to check that these conditions are satisfied:
+
+1. The deadline has been reached.
+2. The bid is higher than the previous bid.
+3. The auction EUTXO is correctly created.
+4. The previous highest bidder gets their bid back.
+
+Transactions for closing the auction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Finally, for this example, let’s assume that there won’t be another bid.
+Once the deadline has passed, the auction can be closed.
+
+In order to do that, somebody has to create another transaction.
+That could be Alice who wants to collect the bid or it could be Charlie who wants to collect the NFT.
+It can be anybody, but Alice and Charlie have an incentive to create it.
+
+This transaction will have one input:
+
+* the auction EUTXO, this time with the `Close` redeemer.
+
+This transaction will have two outputs:
+
+* One output is for the highest bidder, Charlie, who gets the NFT.
+* The second output goes to Alice who gets the highest bid.
+
+In the Close case, the script checks that these conditions are satisfied:
+
+1. The deadline has been reached.
+2. The winner gets the NFT.
+3. The auction owner gets the highest bid.
+
+When there are no bidders
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Finally, we need to consider the scenario in which nobody bids in the auction.
+Alice creates the auction, but the auction receives no bids.
+In this case, there must be a mechanism for Alice to retrieve her NFT.
+
+To address this possibility, Alice creates a transaction with the `Close` redeemer.
+However, because there is no bidder, the NFT doesn’t go to the highest bidder, but instead simply goes back to Alice.
+
+The logic in this case is slightly different.
+It will check that the NFT goes back to Alice.
+However, it doesn’t need to check the recipient because the transaction will be triggered by Alice and she can send the NFT wherever she wants.
```

Comment:
In the previous sentence you said it checks that the NFT goes back to Alice, then you say it doesn't check the recipient, which contradict each other. It _could_ not check that the NFT goes to Alice _if_ it checks that Alice signed the transaction, but you didn't say that.